Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -500,25 +505,33 @@ class ArbitraryMipmapDetector
 
     // This is the average per-pixel, per-channel difference in percent between what we
     // expect a normal blurred mipmap to look like and what we actually received
-    constexpr auto THRESHOLD_PERCENT = 35.f;
+    constexpr auto THRESHOLD_PERCENT = 4.5f;
```

Comment:
Maybe a comment to say the clouds in Galaxy's Observatory are why 4.5% was chosen?

Reply:
Sure, makes sense.